openwisp_controller/geo/estimated_location/test_estimated_location.py

```diff
@@ -440,3 +441,93 @@ def test_estimated_location_notification(self, mock_client, mock_error, mock_inf
             notification.message,
         )
         self.assertIn(device3.last_ip, notification.description)
+
+
+class TestEstimatedLocationApi(TestEstimatedLocationMixin, TestGeoMixin, TestCase):
+    location_model = Location
+    object_location_model = DeviceLocation
+
+    def setUp(self):
+        super().setUp()
+        admin = self._create_admin()
+        self.client.force_login(admin)
+
+    def _create_device_location(self, **kwargs):
+        options = dict()
+        options.update(kwargs)
+        device_location = self.object_location_model(**options)
+        device_location.full_clean()
+        device_location.save()
+        return device_location
+
+    @mock.patch.object(config_app_settings, "WHOIS_CONFIGURED", True)
```

Comment:
Is there a test available where `WHOIS_CONFIGURED` is False and we are verifying that these values are not present?

Reply:
Adding